presentation/matrix_view.py

```diff
@@ -72,69 +78,61 @@ def make_matrix_content(self, results: MeshResults, config: MeshConfig, metric:
-                children=[
-                    html.Span("Time range: "),
-                    html.Span(
-                        "<test_results_timestamp_low>",
-                        className="header-timestamp",
-                        id="timestamp-low",
-                        title=timestamp_low_iso,
-                    ),
-                    html.Span(" - ", className="header-timestamp"),
-                    html.Span(
-                        "<test_results_timestamp_high>",
-                        className="header-timestamp",
-                        id="timestamp-high",
-                        title=timestamp_high_iso,
-                    ),
-                ],
-                className="header__subTitle",
-            ),
-            html.Div(
-                children=[
-                    html.Label("Select primary metric:", className="select_label"),
-                    dcc.Dropdown(
-                        id=self.METRIC_SELECTOR,
-                        options=[{"label": f"{m.value} [{m.unit}]", "value": m.value} for m in MetricType],
-                        value=metric.value,
-                        clearable=False,
-                        className="dropdowns",
-                    ),
-                ],
-                className="select_container",
+            html.Table(
+                className="table-selector-timerange",
+                children=html.Tbody(
+                    children=[
+                        html.Tr(
+                            [
+                                html.Td(
+                                    html.Div(
+                                        children=[
+                                            html.Label("Select primary metric:", className="select_label"),
+                                            dcc.Dropdown(
+                                                id=self.METRIC_SELECTOR,
+                                                options=[
+                                                    {"label": f"{m.value} [{m.unit}]", "value": m.value}
+                                                    for m in MetricType
+                                                ],
+                                                value=metric.value,
+                                                clearable=False,
+                                                className="dropdowns",
+                                            ),
+                                        ],
+                                        className="select_container",
+                                    )
+                                ),
+                                html.Td(
+                                    html.H2(
+                                        children=[
+                                            html.Span("Time range: "),
+                                            html.Span(
+                                                "<test_results_timestamp_low>",
```

Comment:
Non-whitespace placeholder text is undesirable, because it contributes to blinking. I suggest to use:

```
" " * len("00/00/0000, 00:00:00 AM")
```
instead. If there is better way to get the longest possible string for the timestamp text, please use it. BTW, is the 12 hour time format that I'm seeing based on the locale configuration in the browser? As a European bigot (albeit living in USA) I have my system set to 24 hour time (and metric units) and I'm still getting the annoying AM/PM thing (in all browsers). Any idea why is that?

Reply:
Updated in PR#66